http/src/routing.rs

```diff
@@ -485,6 +487,8 @@ pub enum Route<'a> {
         application_id: u64,
         /// The ID of the guild.
         guild_id: u64,
+        /// Whether to include full localization dictionaries.
+        with_localizations: bool,
```

Comment:
Enum structvariants are exhaustive, so this is a breaking change. I would say that if you wanted to add a new field without going onto `next` you could create a new variant, but since we're removing Hash implementations on model types too we'll definitely need to retarget this to `next`.